backends/backend.conf

```diff
@@ -4,6 +4,104 @@ backend {
   default = "Local"
   providers {
 
+    slurm_singularity {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+      config {
+        script-epilogue = "sleep 30"
+        concurrent-job-limit = 50
+        runtime-attributes = """
+          Int cpu = 1
+          Int? time
+          Int? memory_mb
+          String? slurm_partition
+          String? slurm_account
+          String singularity_container
+          String? singularity_options
+          String singularity_command = "exec"
+          String? singularity_command_options
+        """
+        submit = """
+          sbatch \
+          --export=ALL \
+          -J ${job_name} \
+          -D ${cwd} \
+          -o ${out} \
+          -e ${err} \
+          ${"-t " + time*60} \
+          -n 1 \
+          --ntasks-per-node=1 \
+          ${"--cpus-per-task=" + cpu} \
+          ${"--mem=" + memory_mb} \
+          ${"-p " + slurm_partition} \
+          ${"--account " + slurm_account} \
+          --wrap "chmod u+x ${script} && SINGULARITY_BINDPATH=$(echo ${cwd} | sed 's/cromwell-executions/\n/g' | head -n1) singularity ${singularity_options} ${singularity_command} ${singularity_command_options} ${singularity_container} ${script}"
+        """
+        kill = "scancel ${job_id}"
+        check-alive = "squeue -j ${job_id}"
+        job-id-regex = "Submitted batch job (\\d+).*"
+      }
+    }
+
+    sge_singularity {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
```

Comment:
Note that my comments above for slurm apply here too (just translate into sge-land terms).